### ISSUE TITLE
为什么每次从断点处开始训练，学习率会在上一次的基础上减小一半？

### ISSUE THREAD

**Issue by EchoXu98** [Zj-BinXia/ENLCA#13]
Continue from epoch 370...
Making model...
Total params: 41.22M
Preparing loss function:
1.000 * L1
[Epoch 371]	Learning rate: 2.50e-5

这是我训练时打印的部分信息，学习率应该在第400个epoch时减少至2.50e-5，可是这里继续epoch370后学习率直接成了2.50e-5

**Comment by EchoXu98** [comment]
已经解决了
